packages/runtime/src/state/modules/read-only-documents.ts

```diff
@@ -20,24 +20,17 @@ export function createDocumentReference(key: string): DocumentReference {
 export type Document = {
   key: string
   rootElement: Element
+  locale: string | null
```

Comment:
Locale at the document level allows us to have multiple-language sections on the same page.

Reply:
Yes.